### PR TITLE
Hal/qa

### PR DESCRIPTION
**Pull request by hn275**
- user profile clickable, link to account setting page.
- received an optional email from url param for if user is redirected from the landing page, email state is updated accordingly
- controlled form 
- group member transparent background and fixed sizing

### CODE REVIEW

File: src/components/authentication/Login_material.js [hunk 1]
```diff
@@ -502,6 +502,16 @@ function Login (props) {
     updateLoginState(state);
   }
 
+  // set email if redirected from landing page
+  useEffect(() => {
+    const url = new URL(window.location.href);
```
Comment (stephenbelyea) [inline src/components/authentication/Login_material.js:507]: Is the app using React Router or another routing library? If so, we'd have pretty easy [hooks-based access to search params](https://reactrouter.com/en/main/hooks/use-search-params) out-of-the-box.

File: src/components/authentication/Login_material.js [hunk 2]
```diff
@@ -667,8 +680,7 @@ function Login (props) {
                     loadingState={gLoading}
                   />
                 </Grid>
-                {/* <Grid className={`input-box`} style={{ marginTop: '0px' }}> */}
-                {' '}
+                {/* <Grid className={`input-box`} style={{ marginTop: '0px' }}> */}{' '}
```
Comment (stephenbelyea) [inline src/components/authentication/Login_material.js:683]: If we aren't using this commented-out stuff, we should remove it.

**Review by stephenbelyea — APPROVED** [review]
One small note on removing comments and another asking about React Router. Otherwise, looks good. 

Are all the formatting changes (spaces, new lines, etc.) intentional in this PR? If not, it may be good to keep them clear so it's easier to focus on the meat of this one!

File: src/components/authentication/Login_material.js (hunk 1 above)
Comment (hn275) [reply src/components/authentication/Login_material.js:507]: @stephenbelyea the app threw this error when I tried to use the hook
```
Uncaught Error: useLocation() may be used only in the context of a <Router> component.
    at invariant (index.tsx:506:1)
    at useLocation (index.tsx:394:1)
    at useSearchParams (index.tsx:419:1)
    at Login (main.d2b112d9277e8ed326a9.hot-update.js:564:85)
    at renderWithHooks (react-dom.development.js:14982:1)
    at mountIndeterminateComponent (react-dom.development.js:17809:1)
    at beginWork (react-dom.development.js:19047:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3943:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3991:1)
    at invokeGuardedCallback (react-dom.development.js:4054:1)
```

From what I can tell in this [file](https://github.com/TakingITGlobal/commit2act/blob/9adfbe5bb77f6d78217a22cf9b38e6092bf855d1/src/App.js#L105C13-L105C13), this view is conditionally rendered outside of route context provider.

**Comment by hn275** [comment]
> Are all the formatting changes (spaces, new lines, etc.) intentional in this PR? If not, it may be good to keep them clear so it's easier to focus on the meat of this one!

I can see that prettier is configured for this project, do you think we could keep the formatting?

**Pull request merged**